src/main/java/com/gooddata/http/client/GoodDataHttpClient.java

```diff
@@ -81,8 +78,9 @@
     private static final String TOKEN_URL = "/gdc/account/token";
     public static final String COOKIE_GDC_AUTH_TT = "cookie=GDCAuthTT";
     public static final String COOKIE_GDC_AUTH_SST = "cookie=GDCAuthSST";
-    public static final String LOCK_RW = "gooddata.lock.rw";
-    public static final String LOCK_AUTH = "gooddata.lock.auth";
+
+    static final String SST_HEADER = "X-GDC-AuthSST";
```

Comment:
I am not sure about having this not private and using the constant in tests - when somebody change the value of this by mistake, the test won't fail